pod.go

```diff
@@ -239,6 +239,9 @@ type PodConfig struct {
 	AgentType   AgentType
 	AgentConfig interface{}
 
+	NetworkType   NetworkType
```

Comment:
Why ? Don't we want to find back the right network interface handler after a fetchPod() ?